### ISSUE TITLE
Custom preview size

### ISSUE THREAD

**Issue by DasDunkel** [keeporsweep/keeporsweep#33]
It would be nice to be able to configure the size of the preview shown, it can be incredibly hard to see details in images, and since the preview is forced to a square parts can even be missing entirely.

Imo the app should be utilizing the entire viewport to display the image instead of a small section in the center

**Comment by jancborchardt** [comment]
Right, good point. The initial idea was of course to do something like "Tinder for your files/photos", and thus the design was just to look similar. :)

But yes, it makes sense to think about different ways to display it. In the spirit of keeping things simple I would however not make it configurable, but just make a proper new design. If you have any suggestions, feel free to post them! :)

**Comment by ORN-Fox** [comment]
Hi, 

It would be good to modify the display to have a larger rendering. Here is an example that I modify each time locally to have a better view of the images.

<img width="1673" alt="Capture d’écran, le 2024-12-26 à 20 44 44" src="https://github.com/user-attachments/assets/36ef22b0-f858-49b2-9552-5fd1c7ed2463" />
